test/provider/MetadataStore.unittest.ts

```diff
@@ -0,0 +1,79 @@
+import 'jest';
+
+import {Application} from 'openfin/_v2/main';
+
+import {IApplication} from '../../src/client/directory';
+import {AppMetadata, MetadataStore} from '../../src/provider/MetadataStore';
```

Comment:
Flagging that this will conflict - name change of `IApplication`

Reply:
Resolved